pkg/trace/obfuscate/sql_tokenizer.go

```diff
@@ -248,7 +282,10 @@ func (tkn *SQLTokenizer) scanIdentifier() (TokenKind, []byte) {
 	buffer.WriteRune(tkn.lastChar)
 	tkn.next()
 
-	for isLetter(tkn.lastChar) || isDigit(tkn.lastChar) || tkn.lastChar == '.' || tkn.lastChar == '*' {
+	for isLetter(tkn.lastChar) || isDigit(tkn.lastChar) || tkn.lastChar == '*' || tkn.lastChar == '.' {
```

Comment:
Why was `*` and `.` added? Can you please explain? Perhaps an inline comment is nice.